dss/api/search.py

```diff
@@ -26,7 +26,7 @@ def post(json_request_body: dict, replica: str):
         bundles_url_base = request.host_url + 'v1/bundles/'
         result_list = [{
             'bundle_id': hit.meta.id,
-            'bundle_url': bundles_url_base + hit.meta.id.replace(".", "?version=", 1),
+            'bundle_url': bundles_url_base + hit.meta.id.replace(".", "?version=", 1) + f"&replica={replica}",
```

Comment:
Out of the scope if this diff, but this should be using `util.UrlBuilder`

Reply:
Added a TODO to use `util.UrlBuilder`